### PR TITLE
VAS-181 Silabs P10 Project implementation

### PR DESCRIPTION
**Pull request by JulianASLaird**
Support for GATT server and events.
Updated Examples.
Updated bat files for jenkins.

### CODE REVIEW

File: board/filesystem/src/flash_driver.c [hunk 1]
```diff
@@ -18,71 +20,89 @@
...
-  /* Everything worked */
-  return SL_STATUS_OK;
+    xSemaphoreTake(fs_semaphore, portMAX_DELAY);
+	nvm3_ObjectKey_t key_to_erase = block_index + FLASH_DRIVER_BLOCK_KEY_START;
+	Ecode_t error;
+	error = nvm3_deleteObject(NVM3_DEFAULT_HANDLE, key_to_erase);
+//	if (error != ECODE_NVM3_OK) {
+//	    xSemaphoreGive(fs_semaphore);
+//		return SL_STATUS_FAIL;
+//	}
```
Comment (greg-leach) [inline board/filesystem/src/flash_driver.c:103]: Is this needed?

File: examples/Connection/CentralConnection.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    count += 1
+    print(count)
+
+connection.disconnect()
+
+count = 0
+while count < 30:
+    time.sleep(100)
+    count += 1
+#del connection
```
Comment (greg-leach) [inline examples/Connection/CentralConnection.py:35]: Can this go?

File: examples/Connection/PeripheralConnection.py [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import PikaStdLib
```
Comment (greg-leach) [inline examples/Connection/PeripheralConnection.py:1]: Wonder if examples should be pulled out to a separate repo?

File: examples/GATT/GATTIndicate.py [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+            "Read Encryption" : "None",
+            "Write Encryption" : "None",
+            "Capability" : "Indicate",
+            "Callback" : cb_indicate
+        }
+    }
+}
+
+# Setup flag byte array
+flags = [6]
```
Comment (greg-leach) [inline examples/GATT/GATTIndicate.py:77]: Could this be a const?

File: app.c [hunk 5]
```diff
@@ -88,10 +88,7 @@ void python_task(void *plParameters)
 			PikaObj *pikaMain = pikaScriptInit();
 			obj_run(pikaMain, "import PikaStdLib\n"
```
Comment (randyscott) [inline app.c:89]: Is this call to obj_run() with a set of imports actually needed? We don't have anything like this in the Zephyr port.

**Review by randyscott — APPROVED** [review]

**Review by greg-leach — APPROVED** [review]

**Pull request merged**